VocaDbWeb/Scripts/Pages/Album/AlbumBasicInfo.tsx

```diff
@@ -543,12 +542,6 @@ const AlbumBasicInfo = observer(
 						<p>{t('ViewRes.Album:Details.NoTracklist')}</p>
 					)}
 
-				{model.primaryPV && (
-					<div className="song-pv-player">
-						<EmbedPV pv={model.primaryPV} />
-					</div>
-				)}

```

Comment:
Previews must be placed at the top of the page, so that they have a static position. The same goes for events. See also https://github.com/VocaDB/vocadb/issues/1114#issuecomment-1094994750.